src/pybind/mgr/volumes/fs/operations/group.py

```diff
@@ -104,6 +104,16 @@ def create_group(fs, vol_spec, groupname, pool, mode, uid, gid):
     """
     group = Group(fs, vol_spec, groupname)
     path = group.path
+    vol_spec_base_dir = group.vol_spec.base_dir.encode('utf-8')
+
+    # create vol_spec base directory with mode 0o755
+    try:
+        fs.stat(group.path)
+    except cephfs.Error as e:
+        if e.args[0] == errno.ENOENT:
+            fs.mkdirs(vol_spec_base_dir, 0o755)
```

Comment:
maybe set 0o755 as a DEFAULT_MODE class attribute of class VolSpec? What do you think?

Reply:
VolSpec contains other attributes like namespace prefix, isn't it confusing to just have DEFAULT_MODE?

Reply:
I don't think so. Anyways, this suggestion shouldn't block the PR from being merged.

Reply:
Done